mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -223,33 +264,32 @@ def _signed_url_upload_file(self, credentials, local_file):
         except Exception as err:
             raise MlflowException(err)
 
-    def _upload_to_cloud(self, cloud_credentials, local_file, artifact_path):
-        if cloud_credentials.credentials.type == ArtifactCredentialType.AZURE_SAS_URI:
-            self._azure_upload_file(cloud_credentials.credentials, local_file, artifact_path)
-        elif cloud_credentials.credentials.type in [
+    def _upload_to_cloud(
+        self, cloud_credential_info, src_file_path, dst_run_relative_artifact_path
+    ):
+        """
+        Upload a local file to the specified run-relative `dst_run_relative_artifact_path` using
+        the supplied `cloud_credential_info`.
+        """
+        if cloud_credential_info.type == ArtifactCredentialType.AZURE_SAS_URI:
+            self._azure_upload_file(
+                cloud_credential_info, src_file_path, dst_run_relative_artifact_path
+            )
+        elif cloud_credential_info.type in [
             ArtifactCredentialType.AWS_PRESIGNED_URL,
             ArtifactCredentialType.GCP_SIGNED_URL,
         ]:
-            self._signed_url_upload_file(cloud_credentials.credentials, local_file)
+            self._signed_url_upload_file(cloud_credential_info, src_file_path)
         else:
             raise MlflowException(
                 message="Cloud provider not supported.", error_code=INTERNAL_ERROR
             )
 
-    def _download_from_cloud(self, cloud_credential, local_file_path):
+    def _download_from_cloud(self, cloud_credential_info, dst_local_file_path):
```

Comment:
Tweaked some parameter names to be more descriptive and consistent with proto field naming (e.g. `cloud_credentials -> cloud_credential_info` to indicate a single entry within the `credential_infos` proto response field)